tests/simulations/ringtest/create_data.py

```diff
@@ -0,0 +1,158 @@
+#!/bin/env python
+# /// script
+# dependencies = ['h5py', 'libsonata', 'numpy']
+# ///
+# the above allows one to run `uv run create_data.py` without a virtualenv
+
+import h5py
+import libsonata
+import numpy as np
+
+
+def make_nodes():
+    name = "RingA"
+    count = 3
+    with h5py.File("nodes_A.h5", "w") as h5:
+        h5.create_dataset(f"/nodes/{name}/0/model_template", data=["hoc:B_BallStick"] * count)
+        h5.create_dataset(f"/nodes/{name}/0/model_type", data=["biophysical"] * count)
+        h5.create_dataset(f"/nodes/{name}/node_type_id", data=[-1] * count)
```

Comment:
Neurodamus doesn't read these 2 attributes. It might be fine to remove them.

Reply:
`node_type_id` is used by libsonata to determine the size of the population, so we can't remove it.
interesting that `model_type` isn't used; thanks